\documentclass{bredelebeamer}

\renewcommand<>{\item}{\beameroriginal\item\vspace{\stretch{.1}}}
\usefonttheme{professionalfonts} % using non standard fonts for beamer
\usefonttheme{serif} % default family is serif
\usepackage[T1]{fontenc}
\usepackage{lmodern}
\usepackage[]{media9}
\setbeamertemplate{itemize/enumerate body begin}{\Large}
\setbeamertemplate{itemize/enumerate subbody begin}{\large}
\setbeamertemplate{itemize/enumerate subsubbody begin}{\large}

\setbeamertemplate{enumerate item}{%
  \usebeamercolor[bg]{item projected}%
  \raisebox{1.5pt}{\colorbox{bg}{\color{fg}\footnotesize\insertenumlabel}}%
}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\title[Ph.D. Thesis Proposal]{\huge Visualization methods for genealogical and RNA-sequencing datasets}

\author[Lindsay Rutter (ISU)]{\Large Ph.D. Thesis Proposal\\ Lindsay Rutter\\ \vspace{9mm} \normalsize \textbf{Program of Study Committee:}\\ Dianne Cook (Major Professor)\\ Amy Toth (Major Professor)\\ Heike Hofmann\\ Daniel Nettleton\\ James Reecy}


\date[May 16, 2016]{\small May 16, 2016}

\subject{Sujet de votre diaporama}
% C'est utilisé dans les métadonnes du PDF

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}

\begin{frame}
  \titlepage
\end{frame}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% BACKGROUND %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\section{My Background}

\begin{frame}{My Background}

\begin{itemize}
		\item Education
		\begin{itemize}
		  \item B.S. in Bioengineering\\ \textit{Pennsylvania State University} (2003-2007)
		  \item Major in Bioinformatics and Computational Biology\\ \textit{Iowa State University} (2012-Present)
		  \item Minor in Statistics\\ \textit{Iowa State University} (2012-Present)
		\end{itemize}
		\item Internships
		\begin{itemize}
		  \item Okinawa Institute of Science and Technology (Summer 2014)
		  \item MathWorks (Pending - Summer 2016)
		\end{itemize}
	\end{itemize}
\end{frame}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% INTRODUCTION %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 
\section{Introduction}

\begin{frame}{Motivation - Visualizing genealogy}
  \begin{itemize}
		\item ISU agronomists wished to view soybean lineages on the fly
		\item For plant breeding purposes, wanted to display paths between ancestors and descendants constrained by generation for a given variety
		\item Standard pedigree charts could not be used for this application
	\end{itemize}
	\begin{figure}
  \centering
  \fbox{\includegraphics[scale=0.46]{queen.png}}
  \end{figure}
\end{frame}

\begin{frame}{Motivation - Visualizing gene expression}
Cook D, Hofmann H, Lee E-K, Yang H, Nikolau B, Wurtele E. Exploring Gene Expression Data, Using Plots. \textit{Journal of Data Science} 5(2007), 151-182.
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.15]{theTable.png}}
\end{figure}
\begin{itemize}
  \item Mutant organism is defective at synthesizing essential cofactor, which is provided by the treatment
  \item Experiment used 8 Affymetrix GeneChip Arabidopsis Genome Array chips
  \item Each chip has 8297 genes
\end{itemize}
\end{frame}

\begin{frame}
\begin{figure}
\centering
\includegraphics[scale=0.46]{linkPlot.png}
\end{figure}
\end{frame}

\begin{frame}{Motivation - Visualizing gene expression}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.75]{anova.png}}
\end{figure}
\begin{itemize}
  \item Only one gene selected from plots had a low p-value
  \item ANOVA scaled each gene separately, instead of globally
  \item Gene expression analysis best studied using visualizations and models 
\end{itemize}
\end{frame}

\begin{frame}
\begin{figure}
\centering
\includegraphics[scale=0.47]{varPlots.png}
\end{figure}
\end{frame}

\begin{frame}{Motivation - Visualizing gene expression}
\Large{Strive to improve upon five plotting types for RNA-sequencing data:}
\begin{enumerate}
  \item Parallel coordinate plots
  \item Pairwise scatterplots
  \item Replicate line plots
  \item Single gene plots
  \item Permutation testing plots
\end{enumerate}
\end{frame}

\begin{frame}{Thesis proposal overview}
  \begin{itemize}
    \item \textbf{Chapter 2}: Visualizing genealogical data
      \begin{itemize}
        \item Goal: Create unambiguous genealogy visualization plots, adapt genealogical plots for large datasets, incorporate interactive genealogical plots 
      \end{itemize}
    \item \textbf{Chapter 3}: Visualizing cluster analysis of RNA-seq data
      \begin{itemize}
        \item Goal: Develop tools to visualize and interact with gene clusterings to determine genes of interest
      \end{itemize}
    \item \textbf{Chapter 4}: Visualizing significance tests of RNA-seq data
      \begin{itemize}
        \item Goal: Develop tools to visualize, interact with, and permute differentially expressed genes from significance testing
      \end{itemize}
	\end{itemize}
\end{frame}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% GENEALOGY %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Visualizing genealogy}

\begin{frame}
\begin{center}
\fcolorbox{black}{titleColor}{
\begin{minipage}{\textwidth}
\begin{center}
\huge \textbf{Chapter 2:} Visualization methods for\\
genealogical datasets
\end{center}
\end{minipage}
}
\end{center}
\end{frame}

\begin{frame}{Genealogy}
\begin{itemize}
  \item Study of parent-child relationships
  \item Provides tools to better understand traits that arise in lineages
  \begin{itemize}
    \item Desirable (disease resistance)
    \item Undesirable (hemophilia)
  \end{itemize}
  \item Can be represented visually
\end{itemize}
\end{frame}

\begin{frame}{Current visual tools}
\begin{columns}
\begin{column}{0.6\textwidth}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.38]{kinshipFig.png}}
\caption{\texttt{kinship2}: Ambiguous position of green node, who is both second and third generation}
\end{figure}
\end{column}
\begin{column}{0.5\textwidth}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.28]{Graph.png}}
\caption{\texttt{Cytoscape} and \texttt{GraphViz}: Ambiguous position of green node, who is both second and third generation}
\end{figure}
\end{column}
\end{columns}
\end{frame}

\begin{frame}{ggenealogy}
\begin{itemize}
  \item \textbf{ggenealogy}: \texttt{R} package to visualize genealogical structures
  \item First example data is soybean genealogy
  \begin{itemize}
    \item Soybean variety data collected from
      \begin{itemize}
        \item Field trials
        \item Genetic studies
        \item USDA bulletins
      \end{itemize}
    \item Data frame of 412 rows (parent-child relations)
    \item Each variety (n=230)
      \begin{itemize}
        \item Developmental years
        \item Copy number variants (CNV)
        \item Single nucleotide polymorphisms (SNPs)
        \item Protein content and yield
      \end{itemize}
  \end{itemize}
\end{itemize}
\end{frame}

\begin{frame}  
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.59]{LeeAD3.png}}
\end{figure}
\end{frame}

\begin{frame}{Plot shortest path}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.5]{pathTNZB.png}}
\end{figure}
\end{frame}

\begin{frame}{Superimpose path on full structure}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.45]{plotTNBin3.png}}
\end{figure}
\end{frame}

\begin{frame}{Superimpose path on full structure}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.45]{plotTNBin6.png}}
\end{figure}
\end{frame}

\begin{frame}{ggenealogy}
\begin{itemize}
  \item Second example data is genealogy of academic statisticians
  \begin{itemize}
    \item Math Genealogy Project
      \begin{itemize}
        \item Web database of genealogy of academic mathematicians
        \item North Dakota State University Department of Mathematics and the American Mathematical Society
        \item Queried for people with advanced degree in "Statistics" with parent with advanced degree in "Statistics"
      \end{itemize}
    \item Data frame of 8165 rows (3291 parent-child relations)
    \item Each individual (n=7122)
      \begin{itemize}
        \item Year of degree acquisition
        \item Country of degree acquisition
        \item School of degree acquisition
        \item Thesis title
      \end{itemize}
  \end{itemize}
\end{itemize}
\end{frame}

\begin{frame}[fragile]
\frametitle{Plotting ancestors and descendants}
\begin{columns}
\begin{column}{0.5\textwidth}
\begin{semiverbatim}
> statUI <- statGeneal[which(statGeneal
$school=="University of Iowa"),]$child
> length(statUI)
[1] 54
> numDUI <- sapply(statUI, dFunc)
> table(numDUI)
numDUI
 0  1  7 25 
48  4  1  1 
> which(numDUI==25)
Edward Wegman
> which(numDUI==7)
Daniel Nettleton
\end{semiverbatim}
\end{column}
\begin{column}{0.5\textwidth}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.5]{dNett.png}}
\end{figure}
\end{column}
\end{columns}
\end{frame}

\begin{frame}{Superimpose path on full structure}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.5]{DR_All1.png}}
\end{figure}
\end{frame}

\begin{frame}{Superimpose path on full structure}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.5]{DR_All2.png}}
\end{figure}
\end{frame}

\begin{frame}{Animation of path on full structure}
\begin{figure}
    \centering
    \fbox{\includemedia[width=0.9\textwidth, addresource=DRMovie.mov, deactivate=onclick, flashvars={source=DRMovie.mov}]{\includegraphics{./DR_All2.png}}{VPlayer.swf}}
\end{figure}
\end{frame}

\begin{frame}{Future directions}
\begin{itemize}
\item Expand on variable types
\item Enhance interactive capabilities
\item Futher eliminate text overlap
\item Test on larger genealogical datasets
\end{itemize}
\end{frame}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% CLUSTERS %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Visualizing RNA-seq clusters}

\begin{frame}
\begin{center}
\fcolorbox{black}{titleColor}{
\begin{minipage}{\textwidth}
\begin{center}
\huge \textbf{Chapter 3:} Visualization methods for \\
clustering analysis of RNA-sequencing data
\end{center}
\end{minipage}
}
\end{center}
\end{frame}

\begin{frame}{Background of Dataset}
\begin{itemize}
  \item Iron is an essential micronutrient in plants, needed for photosynthesis, metabolism, and respiration
  \item Iron homeostasis is tightly controlled in plants
  \begin{itemize}
    \item Lack of iron leads to iron deficiency chlorosis (IDC)
    \begin{itemize}
      \item Reduction in photosynthesis
      \item Severe nutritional stress
      \item Yield loss
    \end{itemize}
    \item Surplus of iron leads to cellular damage
  \end{itemize}
  \item Plants take up bioavailable iron from soil
  \item IDC is global problem
\end{itemize}
\end{frame}

\begin{frame}{Background of Dataset}
\begin{itemize}
  \item IDC-resistant soybean lines yield lower than susceptible lines in iron-sufficient conditions
  \item Development of IDC-resistant lines with high yield in multiple soil types is ideal
  \item Requires better understanding of genetic basis of iron metabolism in plants
\end{itemize}
\end{frame}

\begin{frame}{Example Dataset (Adrienne Moran Lauter and Michelle Graham)}
\begin{itemize}
  \item Count table for 56,044 genes in 18 samples of soybean leaves
  \begin{itemize}
    \item 30 minutes after iron-rich soil (3 replicates)
    \item 60 minutes after iron-rich soil (3 replicates)
    \item 120 minutes after iron-rich soil (3 replicates)
    \item 30 minutes after iron-deficient soil (3 replicates)
    \item 60 minutes after iron-deficient soil (3 replicates)
    \item 120 minutes after iron-deficient soil (3 replicates)
  \end{itemize}
  \item Count table for 56,044 genes in 18 samples of soybean roots
    \begin{itemize}
    \item 30 minutes after iron-rich soil (3 replicates)
    \item 60 minutes after iron-rich soil (3 replicates)
    \item 120 minutes after iron-rich soil (3 replicates)
    \item 30 minutes after iron-deficient soil (3 replicates)
    \item 60 minutes after iron-deficient soil (3 replicates)
    \item 120 minutes after iron-deficient soil (3 replicates)
  \end{itemize}
\end{itemize}
\end{frame}

\begin{frame}{Motivation}
\begin{itemize}
  \item Cluster genes that show similar changes in read counts across iron condition, time point, and or biological derivative (roots or leaves)
  \item This approach could help us identify what genes are responsible for when and where soybeans respond to iron deficient conditions
  \item Clustering analysis has been applied to plant breeding purposes in previous studies (Newell et al. 2013), although not on RNA-sequencing data
\end{itemize}
\end{frame}

\begin{frame}{Pre-processing}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.4]{needNorm.png}}
\end{figure}
\end{frame}

\begin{frame}{Pre-processing}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.47]{mdsGroups.png}}
\end{figure}
\end{frame}

\begin{frame}{Leaves at 120 minutes}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.47]{L120scatter.png}}
\end{figure}
\end{frame}

\begin{frame}{Leaves at 120 minutes}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.45]{dendL120.png}}
\end{figure}
\end{frame}

\begin{frame}{Leaves at 120 minutes - 2 Clusters}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.34]{pcp2L120.png}}
\end{figure}
\end{frame}

\begin{frame}{Leaves at 120 minutes - 2 Clusters}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.36]{scatterL120.png}}
\end{figure}
\end{frame}

\begin{frame}{Leaves at 120 minutes - 3 Clusters}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.32]{pcp3L120.png}}
\end{figure}
\end{frame}

\begin{frame}{Leaves at 120 minutes - 3 Clusters}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.38]{3scatterL120.png}}
\end{figure}
\end{frame}


\begin{frame}{Leaves at 120 minutes - Many Clusters}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.43]{L120_15.jpg}}
\end{figure}
\end{frame}

\begin{frame}{Leaves at 120 minutes - Many Clusters}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.42]{L120_15_5.jpg}}
\end{figure}
\end{frame}

\begin{frame}{Leaves at 120 minutes - Many Clusters}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.42]{L120_15_6.jpg}}
\end{figure}
\end{frame}

\begin{frame}{Future directions}
\begin{itemize}
\item Render the parallel coordinate plots interactive
\item Adapt parallel coordinate plots to large datasets
\item Develop guidelines for scatterplot matrices
\item Adapt scatterplot matrices to large datasets
\item Render the individual gene plots interactive
\end{itemize}
\end{frame}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% DEGS %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Visualizing RNA-seq DEGS}

\begin{frame}
\begin{center}
\fcolorbox{black}{titleColor}{
\begin{minipage}{\textwidth}
\begin{center}
\huge \textbf{Chapter 4:} Visualization methods for \\
significance testing of RNA-sequencing data
\end{center}
\end{minipage}
}
\end{center}
\end{frame}

\begin{frame}{Background of Dataset}
\begin{itemize}
  \item Phenotypic plasticity is an adaptive tool where one genotype has the ability to produce more than one phenotype when exposed to different biotic and abiotic environment
  \item In Genus \textit{Polistes}, larvae produced early in the colony season (reared by queens) tend to be caste as workers, whereas larvae produced later in the colony season (reared by workers) tend to be caste as queens
  \item Two factors (nutritional and antennal drumming) may be the most likely explanatory variables for the casting fate of \textit{Polistes} offspring
\end{itemize}
\end{frame}

\begin{frame}{Example Dataset (Jennifer Jandt, Ali Berens, Amy Toth)}
\begin{itemize}
  \item RNA-sequencing count tables for 30 samples of \textit{Polistes fuscatus} larvae
  \item Two factors (nutrition and AD) were manipulated to create five treatment groups, each with six samples
  \begin{itemize}
    \item Foundress-Reared ("F") or Worker-Destined
    \item Worker-Reared ("DR", "DU", "NR", "NU") or Queen-Destined
  \end{itemize}
\end{itemize}
\begin{figure}
\centering
\includegraphics[scale=0.59]{groups.png}
\end{figure}
\end{frame}

\begin{frame}{Pre-processing}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.34]{boxplotPW.png}}
\end{figure}
\end{frame}

\begin{frame}{Pre-processing}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.38]{mdsPW.png}}
\end{figure}
\end{frame}

\begin{frame}{Pre-processing}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.3]{scatMatPW.jpeg}}
\end{figure}
\end{frame}

\begin{frame}{Differentially expressed transcripts}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.42]{degPW.png}}
\end{figure}
\end{frame}

\begin{frame}{Future directions}
\begin{itemize}
  \item Strengthen normalization
  \begin{itemize}
    \item Housekeeping transcripts
    \item Internal control transcripts
    \item Spike-in controls
  \end{itemize}
  \item Map to genome
  \item Permutation testing
  \item Render plotting interactive
\end{itemize}
\end{frame}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% TIMELINE %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Timeline}

\begin{frame}{Completed work}
\begin{tabular}{|p{1.5cm}|p{5.5cm}|p{1.8cm}|}
 \hline
 \textbf{Product} & \textbf{Description} & \textbf{Date} \\ 
 \hline
 R package & First release of \texttt{ggenealogy}, which provides visualization tools for genealogical datasets & March 2015 \\
 \hline
 Presentation & Presented \texttt{ggenealogy} at JSM & August 2015 \\
 \hline
 Award & Student paper award at ASA Statistical Computing and Graphics Section & August 2015 \\
 \hline
\end{tabular}
\end{frame}

\begin{frame}{Scheduled deliverables}
\begin{tabular}{|p{1.5cm}|p{5.5cm}|p{1.8cm}|}
 \hline
 \textbf{Product} & \textbf{Description} & \textbf{Date} \\ 
 \hline
 R package & Second release of \texttt{ggenealogy} package, which provides visualization tools for genealogical datasets & May 2016 \\
 \hline
 Paper & Submit \texttt{ggenealogy} paper to JSS & May 2016 \\
 \hline
 R package & First release of package that provides visualization tools for RNA-sequencing datasets & TBD \\
 \hline
 Paper & Submit paper about visualization tools for clustering analysis of RNA-sequencing & TBD \\
 \hline
 Paper & Submit paper about visualization tools for significance testing of RNA-sequencing & TBD \\
 \hline
\end{tabular}
\end{frame}

\begin{frame}{Other work}
\begin{tabular}{|p{1.5cm}|p{5.5cm}|p{1.8cm}|}
 \hline
 \textbf{Product} & \textbf{Description} & \textbf{Date} \\ 
 \hline
 R package & First release of ePort package that generates electronic reports for instructors to evaluate student performance & July 2016 \\
 \hline
\end{tabular}
\end{frame}

\begin{frame}{References}
  \begin{itemize}
    \item \texttt{ggplot2} (Wickham 2009)
    \item \texttt{GGally} (Schloerke et al. 2016)
    \item \texttt{nullabor} (Wickham et al. 2014)
    \item \texttt{ggbio} (Yin et al. 2012)
    \item \texttt{GGobi} (Swayne et al. 2003)
    \item \texttt{tourr} (Wickham et al. 2011)
    \item \texttt{plotly} (Sievert et al. 2016)
    \item Parallel coordinate plots (Inselberg 1985, Wegman 1990)
    \item Visual statistical inference (Chowdhury et al. 2015)
	\end{itemize}
\end{frame}

\begin{frame}{References}
  \begin{itemize}
    \item \texttt{pedigree} (Coster 2013)
    \item \texttt{kinship2} (Therneau et al. 2015)
    \item \texttt{GraphViz} (Gansner and North 2000)
    \item \texttt{Cytoscape} (Shannon et al. 2003)
    \item \texttt{explorase} (Lawrence et al. 2008)
    \item \texttt{edgeR} (Robinson et al. 2010)
    \item \texttt{DESeq2} (Love at al. 2014)
    \item \texttt{RUVseq} (Risso et al. 2014)
    \item Gene expression visual inference (Yin et al. 2013)
    \item Biological clustering: (Newell et al. 2013)
	\end{itemize}
\end{frame}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% EXTRA %%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{frame}{Leaves at 120 minutes - 3 Clusters}
\begin{figure}
\centering
\fbox{\includegraphics[scale=0.65]{indSBGenes2.png}}
\end{figure}
\end{frame}
\end{document}
